debugging.md

```diff
@@ -0,0 +1,68 @@
+# Fehlerbehebung und Debugging
+
+Auch eine REDAXO-Installation kann einmal „Schluckauf“ haben. Im Folgenden werden Möglichkeiten aufgezeigt, deine REDAXO-Installation zu reparieren.
+
+> **Hinweis:** Wir empfehlen auch vor einem Fehler, regelmäßige Backups zu erstellen. Datenbank-Backups können automatisiert über das Cronjob-Addon erstellt werden.
+
+## Fehleranalyse: Whooops und Oooops
+
+![Ooops](/assets/v5.10.0-debug_ooops.png)
+
+Tritt ein Fehler auf, meldet REDAXO sich standardmäßig im Frontend mit einem Oooops und im Backend mit einem Rrrrroar.
+
+![Ooops](/assets/v5.10.0-debug_whooops.png)
+
+Wenn ein Administrator eingeloggt ist, oder der Administrator den  Debug-Modus aktiviert hat, wird anstelle des Oooops ein Whoooops mit genauerer Fehlerbeschreibung und Stacktrace ausgegeben.
+
+## Fehleranalyse: Die system.log-Datei
+
+![Screenshot /System/Logdateien](/assets/v5.10.0-debug_syslog.png)
+
+In der Datei `redaxo/data/core/system.log` werden Fehler geloggt - möglicherweise ist der Fehler bereits dabei.
+
+## Debugging: Einstellungen in der `config.yml`
+
+In der Datei `redaxo/data/core/config.yml` können Parameter zum Debugging aktiviert werden. Der Debug-Modus sollte ausschließlich nur dann aktiviert werden, wenn die Website nicht öffentlich erreichbar ist.
+
+## Debugging: Das debug-Addon
+
+![Debug-AddOn ab REDAXO 5.11](/assets/v5.11.0-debug_addon.png)
+
+Neu in REDAXO 5.11 ist das Debug-Addon, welches im Backend zusätzlich aktiviert werden kann. Es erscheint ein neuer Menüpunkt, in dem Clockwork gestartet wird. Jeder weitere Aufruf im Frontend oder Backend übergibt an Clockwork Debugging-Informationen.
+
+## Debugging: Entwicklertools des Browsers
+
+![Safari Netzwerk/Serververhalten](/assets/v5.10.0-browser_network.png)
+
+In modernen Browsern befindet sich in den Entwickler-Tools ein "Netzwerk"-Reiter, in welchem REDAXO bei aktiviertem Debug-Modus zusätzliche Performance-Informationen ausgibt, die über `rex_timer` gemessen werden.
+
+## Fehler: Google indexiert die Seite nicht, alle Seiten werden als noindex markiert.
+
+Möglicherweise ist der Debug-Modus aktiviert. Der Debug-Modus sollte ausschließlich nur dann aktiviert werden, wenn die Website nicht öffentlich erreichbar ist.
+
+## Fehler: REDAXO funktioniert nach einem Website-Umzug nicht mehr
+
+* Ist sichergestellt, dass alle Dateien erfolgreich kopiert wurden?
+* Stimmt die PHP-Version?
+* Wurden die Datenbank-Zugangsdaten angepasst?
+* Sind alle Einstellungen am Paket korrekt, z.B. Domain, A-Record, Verzeichnispfad, etc.?
+* Wurde die .htacess-Datei übernommen oder neu gesetzt oder sind Anpassungen dort nötig?
+
+## Fehler: Ich kann mich nicht mehr einloggen
+
+Möglicherweise wurde das Passwort geändert oder der Benutzer existiert nicht mehr. Oder: Möglicherweise wurde ein Backup eingespielt, das die Tabelle `rex_user` nicht beinhaltet hat oder überschrieben hat. 
+
+* Lösung 1: Ggf. das Setup starten, um den Administrator erneut anzulegen.

```

Comment:
Wie startet man das Setup?

Reply:
Gute Frage, wurde das bisher nie erklärt? Wo sollte ich das unterbringen?

Reply:
Setze doch einfach einen Link zu https://redaxo.org/doku/master/passwort-vergessen 
Ich würde den Weg über das Setup so nicht empfehlen, da die Gefahr besteht auch noch den Rest der Seite vollständig zu löschen

Reply:
Ok, das verbessern wir in der nächsten Iteration. Ich mache mir dazu mal Gedanken. Einverstanden?